tests/PropertyList.py

```diff
@@ -17,17 +18,16 @@
 # 
 # You should have received a copy of the LSST License Statement and 
 # the GNU General Public License along with this program.  If not, 
-# see <http://www.lsstcorp.org/LegalNotices/>.
+# see <https://www.lsstcorp.org/LegalNotices/>.
 #
 
 import unittest
-import pickle
+import cPickle as pickle
```

Comment:
Not sure we can get around it though. The standard pickle has a problem because it tries to call `object.__new__` to create the object which bypasses the C++ memory allocation. Fixing this requires the `__getstate__` / `__setstate__` callback to Python trick. But these are skipped when using pickle or protocol version 1 (see below).